src/commands/load-all.build.ts

```diff
@@ -226,6 +226,10 @@ function determineFunctionFlags(f: Builder.ParsedFunction) {
     flags.push("DoNotReplay");
   }
 
+  if ("keepephemeral" in f.properties) {
```

Comment:
"keepephemeral" has the same casing as other JSDoc tags but is pretty hard to read IMO (especially because of all these p's and e's close together). A couple of alternative ideas: keepcontext, preserve, preservecontext, noreset, noresetstate. I have a slight preference for `keepcontext` as my brain can easily separate both words, but don't have a strong opinion.